notebooks/en/_toctree.yml

```diff
@@ -61,4 +61,9 @@
 - title: Agents
   sections:
   - local: agents
-    title: Build an agent with tool-calling superpowers using Transformers Agents
\ No newline at end of file
+    title: Build an agent with tool-calling superpowers using Transformers Agents
+
+- title: Enterprise Hub Cookbook
```

Comment:
I wonder if it might be better to call it "Enterprise Hub recipes" instead of a cookbook? I think it can be a bit confusing that there is another "cookbook" inside the open-source cookbook 😅